sdk/azidentity/bearer_token_policy_test.go

```diff
@@ -174,15 +174,22 @@ func TestBearerPolicy_GetTokenFailsNoDeadlock(t *testing.T) {
 		HTTPClient:    srv,
 		AuthorityHost: srv.URL(),
 		Retry: azcore.RetryOptions{
-			// leaving TryTimeout at zero will trigger a deadline exceeded error causing GetToken() to fail
-			RetryDelay:    50 * time.Millisecond,
-			MaxRetryDelay: 500 * time.Millisecond,
-			MaxRetries:    3,
+			// leaving TryTimeout at as a very small value will trigger a deadline exceeded error causing GetToken() to fail
+			TryTimeout: 1 * time.Nanosecond,
+			MaxRetries: 0,
```

Comment:
Missing original values for `RetryDelay` and `MaxRetryDelay`.  Also, the zero here is redundant (doesn't hurt anything of course).

Reply:
Done and fixed, it was supposed to be a -1 so it doesnt retry.